op-node/rollup/types.go

```diff
@@ -63,6 +63,11 @@ type AltDAConfig struct {
 	DAResolveWindow uint64 `json:"da_resolve_window"`
 }
 
+// DAWindow is number of L1 blocks to wait for challenge and resolve in the worst case, including the challenged L1 origin block itself
+func (a *AltDAConfig) DAWindow() uint64 {
+	return a.DAChallengeWindow + a.DAResolveWindow + 1
```

Comment:
```suggestion
	return a.DAChallengeWindow + a.DAResolveWindow
```

Reply:
How about we remove the +1 from here, and add it in the finalizer instead? I feel like this way the definition of DAWindow() here aligns with the defns inside the contract, and we also don't impact the defn on SyncLookback().

Reply:
But then it doesn't align with the definition of `SeqWindowSize`, which is inclusive.

Reply:
That just boils down to your interpretation and use of the variable. In the contract for eg they use < n instead of <= n+1

Reply:
The contract is using `<=`: https://github.com/ethereum-optimism/optimism/blob/0d3273e267802599399ae99426a44a8dedfd8444/packages/contracts-bedrock/src/L1/DataAvailabilityChallenge.sol#L209 and https://github.com/ethereum-optimism/optimism/blob/0d3273e267802599399ae99426a44a8dedfd8444/packages/contracts-bedrock/src/L1/DataAvailabilityChallenge.sol#L216 .

curious what other guys will think:)

Reply:
ehh good catch not sure what I was reading haha. I think in either case we want it to be inclusive (like the <= in the contract). The question is whether we define the variable to be n+1, or keep the definition the same and add the +1 in the lookback (which is ultimately just used to define the length of an array to store blocks that we need to look back to).

I would vote to keep the DAWindow defn same as in the contract, and add the +1 manually where needed.